pkg/goDB/engine/query.go

```diff
@@ -320,6 +330,9 @@ func (qr *QueryRunner) RunStatement(ctx context.Context, stmt *query.Statement)
 			count++
 		}
 
+		// add statistics to final result
+		result.Summary.Stats.Add(aggMap.Stats)
```

Comment:
Given that it's in the AggFlowMapWithMetadata, we can now propagate the stats all the way up to the result